cookiecutter/zipfile.py

```diff
@@ -0,0 +1,123 @@
+from __future__ import absolute_import
+
+import os
+import requests
+import tempfile
+from zipfile import ZipFile
+try:
+    from zipfile import BadZipFile
+except ImportError:
```

Comment:
Can you please add a comment mentioning that ``BadZipfile`` was deprecated in Python 3.2?